wandb/integration/tensorboard/log.py

```diff
@@ -294,4 +294,5 @@ def log(tf_summary_str_or_pb, history=None, step=0, namespace="", **kwargs):
         ):
             history.add({}, **kwargs)
         STEPS["global"]["last_log"] = timestamp
+    log_dict["_step"] = STEPS["global"]["step"]
```

Comment:
This makes me a little nervous.  Are we sure we need this?

Reply:
Its not great. I think another option would be to have the sync manager just inject the _step metric as global_step? Then we could drop the other change you're nervous about to

Reply:
I'll leave it up to you.  My fear is having this come through so many different paths will make future bugs more likely.

Reply:
Totally agree. let me test that out and see if it works. If it does, I like it more. Otherwise I'll play around with finding another work around